### DISCUSSION TITLE
link rel="preload" does not include cookie

### DISCUSSION THREAD

**Discussion by safinn** [vercel/next.js#13370] [Help] [answered]
I'm trying to using a <link> tag to preload data as recommended in the SWR package but the request that is made does not contain my cookie so fails every time. My cookie has `secure = true`, `SameSite = Lax` and `HttpOnly = false`. The cookie domain is set to the api which is on a subdomain of the next app (this may be the issue, not sure).

How do I include the cookie in the request made by `<link rel="preload" href={`${process.env.NEXT_PUBLIC_API_URL}/user`} as="fetch" crossorigin="anonymous" />`?

**Comment by patlux** [comment, marked as answer]
Remove `crossorigin="anonymous"`. See here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#attr-crossorigin
